Makefile

```diff
@@ -53,18 +41,43 @@ katana-genesis: install-katana
 run-katana: katana-genesis
 	katana --disable-fee --chain-id=kkrt --genesis .katana/genesis.json
 
+# Total test suite.
 test: katana-genesis load-env
 	cargo test --all --features testing
 
-# Make sure to have a Kakarot RPC running and the correct port set in your .env and an underlying Starknet client running.
-benchmark-madara:
-	cd benchmarks && bun i && bun run benchmark-madara
-
+# Run a specific test target. Need to run make katana-genesis once first.
+# Example: `make test-target TARGET=test_raw_transaction`
 test-target: load-env
 	cargo test --tests --features "testing,hive" $(TARGET) -- --nocapture
 
-benchmark-katana:
+benchmark:
 	cd benchmarks && bun i && bun run benchmark-katana
```

Comment:
did the benchmark got unified for madara and katana? if so, you should also change the name on the npm script to `benchmark`